### ISSUE TITLE
Option to add GMT Date as sidebar info (ex: 2016.243)

### ISSUE THREAD

**Issue by Eyeheartpie** [freakified/TimeStylePebble#84]
Not a huge demand I imagine, but being able to show calendar date as well as GMT date would be helpful for me.

**Comment by tilden** [comment]
You can't really fit more than 3 characters, so would this basically amount to the day number (like in issue #50)?

**Comment by Eyeheartpie** [comment]
Yeah, I think that would work. Just 3 characters where the date would otherwise be. I think I can usually remember what year it is.
